website/src/word-panel.tsx

```diff
@@ -95,12 +95,9 @@ export const WordPanel = (p: {
           title={"Word Parts"}
           content={
             <>
-              <MorphemicSegmentation
-                segments={p.segment.segments}
-                onOpenDetails={p.onOpenDetails}
-                level={p.viewMode}
-                tagSet={p.tagSet}
-              />
+              <table className={css.table}>
+                <PartLines segments={p.segment.segments} />
```

Comment:
Can we call the component something more descriptive? For example, `VerticalMorphemicSegmentation` works. It's long but the name is more accurate. Also, move the `table` element inside that component and use it to replace the plain `div` you've got there now. General rule of thumb: if you have a `<div>` with no attributes and it's the only child of its parent element, then you don't need it at all.